certified-connectors/Tesseron Invoice/readme.md

```diff
@@ -0,0 +1,41 @@
+## Tesseron ASM Ticket Connector
+With Tesseron, we offer your company an innovative solution for cooperation with your customers, partners and suppliers.This connector allows you to use Tesseron Invoice API to create position notes and activity recordings.
+
+## Publisher: Publisher's Name
+Lutihle + Luithle GmbH
+
+
+## Prerequisites
+You will need the following to proceed:
+* A Tesseron Instance
+* A Tesseron licensed user
+* An API Key (Service: Invoice)
+
+
+## Supported Operations
+The connector supports the following operations:
+* `Create Inovice Position Note`: Create a position note with or without tickets
+* `Create Activity Recording`: Add a new activity recordings to your profile
+
+
+## Obtaining Credentials
+Authentication is done via an API key. Please provide your API key and your instance URL als connection parameters.
+
+
+## Known Issues and Limitations
+* Sufficient user rights are mandatory to receive ticket information in specific areas
+* If the ticket creation is triggered multiple time at once there may be an incomplete ticket creation
```

Comment:
@developmentLL all files should be free of grammatical and syntax errors. here you have "...multiple time...", Maybe you meant to say "...multiple times...".